database/db.js

```diff
@@ -31,13 +31,69 @@ if (process.env.NODE_ENV === 'test') {
         if(summary[0] === undefined) {
           cb(new Error('Could not find record in database'));
         } else {
+          console.log('here', summary[0]._doc);
           cb(null, summary[0]._doc);
         }
       }
     })
   }
 }
 
+let insertNewSummary = (info, cb) => {
+  // need to make sure stayId isn't already taken by another room
+  let newStayId = info.stayId;
+  let numBeds = info.numBeds;
+  let numBedrooms = info.numBedrooms;
+  let numBaths = info.numBaths;
+  let numGuests = info.numGuests;
+  let typeOfStay = info.typeOfStay;
+
+
+  StaySummary.find({stayId: newStayId}, (err, summary) => {
+    if (err) {
+      console.log('Record already in database');
+      cb(err, 404);
+    } else if (summary.length > 0) {
+      cb(err, 404);
+    } else {
+      StaySummary.create({stayId: newStayId, numBeds: numBeds, numBedrooms: numBedrooms, numBaths: numBaths, numGuests: numGuests, typeOfStay: typeOfStay}, (err, result) => {
```

Comment:
if you name all variables as the same as property name you can just pass an object like `({stayId, numBeds, .. etc})`